corehq/apps/reports/standard/monitoring.py

```diff
@@ -1453,7 +1461,16 @@ def get_admins_and_demo_users(self):
     @property
     def users_to_iterate(self):
         if toggles.EMWF_WORKER_ACTIVITY_REPORT.enabled(self.request.domain):
-            user_query = EMWF.user_es_query(self.domain, self.request.GET.getlist(EMWF.slug))
+            user_query = LocationRestrictedMobileWorkerFilter.user_es_query(
+                self.domain, self.request.GET.getlist(LocationRestrictedMobileWorkerFilter.slug)
+            )
+            if not self.request.couch_user.has_permission(self.domain, 'access_all_locations'):
+                accessible_location_ids = (SQLLocation.active_objects.accessible_location_ids(
+                    self.request.domain,
+                    self.request.couch_user)
+                )
+                accessible_user_ids = user_ids_at_locations(accessible_location_ids)
+                user_query.filter(accessible_user_ids)
```

Comment:
Oh right, just saw @gcapalbo 's comment - I think you can drop the `user_ids_at_locations` call here and just do:
```python
user_query = user_query.location(accessible_location_ids)
```
The `filter` method you're using here expects a json filter, and it (along with the other filter methods) doesn't mutate the query - it returns a new query, so you need to assign it to something.